src/libs/antares/writer/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+project(result-writer)
+add_library(result_writer
+        # Interface class (public API)
+        antares/writer/i_writer.h
+        antares/writer/result_format.h
+
+        # Helper class
+        private/ensure_queue_started.h
```

Comment:
We don't need the "private" directory, because the public headers are anyway already separated in their own directory.

Reply:
Public headers are in their own directory but target_include_directories specifies the top "writer" directory as public, making all others headers (and sources ) visible. I added "private" to prevent unwanted inclusion.

Will be solved in the future by making a "include" intermediate directory